envi/config.py

```diff
@@ -80,6 +80,48 @@ def getOptionDoc(self, optname):
         '''
         return self.cfgdocs.get(optname)
 
+    def getConfigPaths(self, with_vals=False):
```

Comment:
not a fan of the with_vals option which basically makes it so that the api can return two differently structured result sets.  any reason not to just always include the values? ( read the next comment before answering this one ;) )